config/routes.rb

```diff
@@ -1,8 +1,7 @@
 Rails.application.routes.draw do
-  get 'pages/hello'
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
-  root 'pages#hello'
+  get 'home/home'
+  root 'home#home'
+  get "/users/:id", to: "users#show"
+  get "/users/:id/posts", to: "users#posts"
+  get "/users/:id/posts/:id", to: "users#post"
```

Comment:
- [ ] We need to have an endpoint for users index action which renders all users. See the suggestion below;

- [ ] Since we have deleted the `posts` and `post` actions in the UsersController, we do not need them here anymore, instead we add the `index` and `show` actions from PostsController. 

```suggestion
  get "/users", to: "users#index"
  get "/users/:user_id", to: "users#show"
  get "/users/:user_id/posts", to: "posts#index"
  get "/users/:user_id/posts/:id", to: "posts#show"
```


- [ ] **[OPTIONAL]** Another pain-free and easier way of creating endpoints is by the use of resources, check out this [article](https://guides.rubyonrails.org/routing.html#configuring-the-rails-router) to see how to use it. It will look something like this;

```ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```